File: Java/HissingMicrophone.java
/*
* Problem: https://open.kattis.com/problems/hissingmicrophone
* 
* @author  Bivash Pandey
*/

import java.util.Scanner;

public class HissingMicrophone {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String word = input.next();
        String ans = "no hiss";
        for(int i=0; i<word.length()-1; i++) {
            if(word.charAt(i)=='s' && word.charAt(i+1)=='s') {
                ans = "hiss";
                break;
            }
        }
        System.out.println(ans);
    }
}

File: Python/HissingMicrophone.py
'''
Problem: https://open.kattis.com/problems/hissingmicrophone
Author:  Bivash Pandey
'''
word = input()
ans = "no hiss"
for i in range(len(word)-1):
    if word[i] == 's' and word[i+1] == 's':
        ans = "hiss"
        break
print(ans)